### DISCUSSION TITLE
SIWE Sign in with Ethereum

### DISCUSSION THREAD

**Discussion by zdanl** [ethereum/ethereum-org-website#11793] [General]
cheerio.
i'm setting up a thing and have been looking into supabase type things.
SIWE is clearly the new hotness in web2 migration, but we should migrate legacy users as well.
Brave once promised .crypto domains to be available without DNS. Never happened.
window.ethereum doesn't default to Brave Wallet, which I understand, also Brave isn't legacy.

Chrome would be. Edge too. Same thing.
Can we make a preliminary wallet, that UI wise is not meant for financial transactions without one-click, autogenerated, private key and seedphrase in chome://crypto, so that we don't need Browser extensions anymore, for new users?

Specifically, for SIWE to just work..

thoughts or hate welcome.

**Comment by corwintines** [comment]
Big supporter of SIWE, but this doesn't seem related to ethereum.org, and is a whole different product. Closing this out.
